src/modules/dashboard/components/ExpenditurePage/Streaming/types.ts

```diff
@@ -1,15 +1,34 @@
+import { DatePickerFieldValue } from '~core/Fields/DatePicker/DatePicker';
+import { AnyUser } from '~data/index';
+import { ExpenditureEndDateTypes } from '~pages/ExpenditurePage/types';
+
+export interface Rate {
+  amount?: number;
+  token?: string;
+  time?: string;
+  limit?: number;
+  id: string;
+}
+
 export interface FundingSource {
   team: string;
-  rate: {
-    amount?: number;
-    token?: string;
-    time?: string;
-  };
+  rate: Rate[];
```

Comment:
A small suggestion to rename this to `rates` to indicate it's an array of rates rather than a single rate